content/en/blog/kotlin-gradle-projects.md

```diff
@@ -9,13 +9,16 @@ author:
 
 Did you know that gRPC Java now has out of box support for Kotlin projects built with Gradle? [Kotlin](https://kotlinlang.org/) is a modern, statically typed language developed by JetBrains that targets the JVM and Android. It is generally easy for Kotlin programs to interoperate with existing Java libraries. To improve this experience further, we have added support to the [protobuf-gradle-plugin](https://github.com/google/protobuf-gradle-plugin/releases) so that the generated Java libraries are automatically picked up by Kotlin. You can now add the protobuf-gradle-plugin to your Kotlin project, and use gRPC just like you would with a typical Java project.
 
-<!--more-->
+{{% alert title="Native Kotlin support" color="success" %}}
+Looking for native Kotlin support of gRPC? See [Kotlin, meet gRPC]({{<relref "kotlin-meet-grpc">}}).
+{{% /alert %}}
+
 
 The following examples show you how to configure a project for a JVM application and an Android application using Kotlin.
 
 ### Kotlin gRPC client and server
 
-The full example can be found [here](https://github.com/grpc/grpc-java/tree/master/examples/example-kotlin).
+The full example can be found [here](https://github.com/grpc/grpc-java/tree/v1.29.0/examples/example-kotlin).
```

Comment:
This doesn't use gRPC Kotlin. You could point to the gRPC Kotlin examples instead: https://github.com/grpc/grpc-kotlin/tree/master/examples

Reply:
Ah, I think I see now that you are just try to preserve the old stuff and have the alert send them to the new stuff.

Reply:
@jamesward - Right. This is an old 2018 blog post about gRPC Java's support for Kotlin. I want to keep the links to the old examples just for consistency with the rest of the blog post. But that is why I added the alert. Does that make sense?

Reply:
Oh, the posting of our comments crossed!

Reply:
So, is this good to go?